curriculum/README.md

```diff
@@ -10,9 +10,10 @@ This curriculum is based on and aligned with Professor Stuart Reges' course at t
 Washington, CSE 142. The course uses the textbook _Building Java Programs: A Back to Basics
 Approach_, by Stuart Reges and Marty Stepp. The course is aligned with the AP Computer Science A
 standards. TEALS has received AP Audit certification for previous versions of the course and
-syllabus. Since these materials are new in 2015–2016, TEALS will apply to have the new syllabus
-approved. Once the CollegeBoard approves the new syllabus, partner schools may use the “claim
-identical” function of the AP Audit website to obviate the need for their own curriculum audit.
+syllabus.
+
+The TEALS AP curriculum was approved by the CollegeBoard so partner schools may use the “claim
+identical” function of the AP Audit website to obviate the need for their own curriculum audit. Specific instructions are available [here].
```

Comment:
Please keep lines to 100 characters in length max. Markdown files should be legible as plaintext in addition to rendered in browser.

Reply:
Change "[here]" to "in the [AP CS A Course Audit Instructions] document."